docs/apm/correlations.asciidoc

```diff
@@ -21,53 +19,60 @@ Or, perhaps set of users, based on IP address or region, is facing increased lat
-Correlated attributes are sorted by _Impact_–a visual representation of the
-{ref}/search-aggregations-bucket-significantterms-aggregation.html[significant terms aggregation]
-score that powers correlations.
-Attributes with a high impact, or attributes present in a large percentage of slow transactions,
-may contribute to increased latency.
-
-To find high-latency correlations, hover over each potentially correlated attribute to
-compare the latency distribution of transactions with and without the selected attribute.
-
-For example, in the screenshot below, the field `user_agent.name` and value `HeadlessChrome`
-exists primarily in higher-latency transactions between 3.7 and 8.7 seconds.
+The correlations on the *Latency* tab help you discover which attributes are
+contributing to increased service latency.
 
 [role="screenshot"]
-image::apm/images/correlations-hover.png[Correlations hover effect]
-
-Select the `+` filter to create a new query in the APM app for transactions with
-`user_agent.name: HeadlessChrome`. With the "noise" now filtered out,
+image::apm/images/correlations-hover.png[Latency correlations]
+
+A latency distribution chart visualizes the overall latency of the service. If
+there are attributes that have a statistically significant impact on the
+latency, they are listed in a table below the chart. The table is sorted by
+correlation values, which are Pearson correlation coefficients (PCC) that range
+from 0 to 1. Attributes with higher correlation values are more likely to
+contribute to high latency transactions. By default, the attribute
+with the highest correlation value is added to the chart.
+
+To find high-latency correlations, hover over each potentially correlated
+attribute to compare the latency distribution of transactions with and without
+the selected attribute.
+//TBD: Talk about interpreting the skew of attributes in chart?
+
+If a correlated attribute seems noteworthy, use the **Filter** quick links:
+
+* `+` creates a new query in the APM app for filtering transactions containing
+the selected value.
+* `-` creates a new query in the APM app to filter out transactions containing
+the selected value.
+
+In this example screenshot, the field `labels.orderPriceRange` and value `large`
+exists primarily in higher-latency transactions between 0.1 and 4.0 seconds.
+Select the `+` filter in the first row of the table to create a new query in the
+APM app for transactions with this attribute. With the "noise" now filtered out,
 you can begin viewing sample traces to continue your investigation.
 
-As you sift through high-latency transactions, you'll likely notice other interesting attributes.
-Return to the correlations fly-out and select *Customize fields* to search on these new attributes.
-You may need to do this a few times–each time filtering out more and more noise and bringing you
-closer to a diagnosis.
-
 [discrete]
 [[correlations-error-rate]]
 ==== Find error rate correlations
```

Comment:
Should we call this 'failed transaction correlations` rather than `error rate correlations`?

Reply:
I've made that change in 953633c4ba7c8ab1df772d483537a753c24ea4a3 (and altered the page title too)